c++/src/capnp/rpc.c++

```diff
@@ -3081,15 +3081,15 @@ public:
   Impl(VatNetworkBase& network, kj::Maybe<Capability::Client> bootstrapInterface)
       : network(network), bootstrapInterface(kj::mv(bootstrapInterface)),
         bootstrapFactory(*this), tasks(*this) {
-    tasks.add(acceptLoop());
+    acceptLoopPromise = acceptLoop().fork();
```

Comment:
We should add an `.eagerlyEvaluate()` with a callback that logs the exception... in order to maintain the existing behavior for anyone who doesn't update their code to call `getAcceptLoopPromise()`.